### ISSUE TITLE
Missing author names in AP => wm replies

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy-fed#452]
[Example](https://fed.brid.gy/render?id=https%3A%2F%2Ftech.lgbt%2Fusers%2Fnelson%2Fstatuses%2F110028718037060258), [log](https://fed.brid.gy/log?start_time=1678905013&key=https://tech.lgbt/users/nelson/statuses/110028718037060258/activity&module=):

```html
  <span class="p-author h-card">
    <data class="p-uid" value="https://tech.lgbt/users/nelson"></data>
    <a class="u-url" href="https://tech.lgbt/users/nelson">tech.lgbt/users/nelson</a>
  </span>
```

Was `object.author` maybe a string id before, so we'd fetch and hydrate it, but it's now an object, so we don't?

```json
  "object": {
    "author": {
      "id": "https://tech.lgbt/users/nelson"
    }
    "...",
```

**Comment by snarfed** [comment]
The stored `Object` for https://tech.lgbt/users/nelson/statuses/110028718037060258 has `as1` as above, ie `object` is a composite object, but `as2` has bare string `"attributedTo": "https://tech.lgbt/users/nelson"`, ie not composite object.

**Comment by snarfed** [comment]
Fixed in 0ba43561fa7784d72d577d1471a9b50f7d853992
